editors/code/package.json

```diff
@@ -39,11 +39,13 @@
         "package": "vsce package -o clarity-lsp.vsix",
         "compile": "tsc -b",
         "build": "tsc",
+        "build:syntax": "js-yaml syntaxes/clarity.yaml > syntaxes/clarity.tmLanguage.json",
```

Comment:
could we add this this step to the `build` step?

Reply:
Merged `build:syntax` with `build`